client/store/singleProduct.js

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios'
```

Comment:
I don't know if we need this file to view a single product. It can wait until tomorrow when we're discussing it on a call though.

Reply:
yeah i'm pretty sure prof said something about not doing this , even though it would still work it is not necessary

Reply:
I have revised the single products files accordingly. Please take a look and merge when you get a chance. Thanks.